udata/core/reuse/models.py

```diff
@@ -105,6 +105,8 @@ def post_save(cls, sender, document, **kwargs):
             cls.on_create.send(document)
         else:
             cls.on_update.send(document)
+        if document.deleted:
```

Comment:
Yes, the update is sent on any modification. Indexing/Deindexing listen on `on_update`